Userland/Services/SpiceAgent/ChunkHeader.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2023, Caoimhe Byrne <caoimhebyrne06@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Forward.h>
+#include <AK/Types.h>
+
+namespace SpiceAgent {
+
+class ChunkHeader {
+public:
+    // Indicates where the message has come from
+    enum class Port : u32 {
+        Client = 1,
+
+        // There are currently no messages which are meant for the server, so all messages sent by the agent (us) with this port are discarded.
```

Comment:
This comment should presumably be located whereever we are discarding those messages.

Reply:
The spice server does, which is what this comment refers to.